js/components/CategoriesItem.jsx

```diff
@@ -67,9 +66,7 @@ var CategoriesItem = React.createClass({
             <Link to="category" params={{category: this.props.category.name}}>{this.props.category.name}</Link>
             {deleteBtn}
           </td>
-          <td className="percentile-col"><ScoreBar percentile={this.props.category.percentile}
-            previousPercentile={this.props.category.previousPercentile} category={this.props.category.name}/>
-          </td>
+          <td>{this.props.category.rank}</td>
```

Comment:
Temporarily getting rid of the scorebar. I think it'll be cool to replace with something else anyway. Maybe a colored arrow that goes up or down